docs/src/getstarted/overview.md

```diff
@@ -0,0 +1,140 @@
+---
+title: "Introduction to Solana Development"
+description: "Learn about the basic development concepts of the Solana blockchain."
+keywords:
+  - accounts
+  - transactions
+  - nft
+  - solana basics
+  - tutorial
+  - intro to solana development
+  - blockchain developer
+  - blockchain tutorial
+  - web3 developer
+---
+
+Welcome to the Solana developer docs!
+
+This guide contains step-by-step instructions on how to get started. Before we get into the hands on part of the guide, we'll cover basic concepts that all developers need to be familiar with to build on Solana:
+- Transactions
+- Accounts
+- Programs
+
+## What you will learn
+- What the developer workflows look like
+- What transactions, accounts, and programs are
+- Test networks and other tools
+
+## An overview of Solana developer workflows
+The Solana network can be thought of as one massive global computer where anyone can store and execute code for a fee. Deployed code is called a program, often referred to as a "smart contract" on other blockchains. To interact with a program, you need to send a transaction on the blockchain from a client.
+
+Here's a high level representation of this. It’s important to note that this is an oversimplification of the Solana network for the purposes of learning in an easy-to-understand way.
+
+![Solana developer workflows program-client model](https://lh5.googleusercontent.com/Z1UcGnLLqEeCTWLm0QjMscPfFKhRumOcuFc8uadpn7wqO7OlsQiNe60CpGsPZf1zjsLTpj-69IeQf9MwCdMnFdyyDNkI8x0uPHPyPf5DUX_uqKWYuJq7o5p9TZ5_TqpFCO7ZS6pO)
+
+**Program development**  
+The first development workflow allows you to to create and deploy custom Rust, C and C++ programs directly to the blockchain. Once these programs are deployed, anyone who knows how to communicate with them can use them. 
+
+You can communicate with these programs by writing dApps with any of the available client SDKs (or the CLI), all of which use the JSON RPC API under the hood.
```

Comment:
can you make the text for cli and json rpc api to their respective internal link pages